app/views/admin/offences/_form.html.erb

```diff
@@ -1,115 +1,158 @@
-<div class="nested-fields">
+<%= form_with(model: offence, url: url, local: true) do |f| %>
```

Comment:
I thought that this would fail more gracefully, but when not filling some items in the form I got:
Validation failed: Offencedefinitionid can't be blank, Offencetitle can't be blank

Reply:
More gracefully in what way?

If we were using snake case, i.e. `offence_definition_id`, Rails would replace `_` with spaces, and we'd get a prettier validation error message.

Unfortunately we're going counter to convention and using camel case.

Reply:
Didn't see this come through, sorry! This is fixed now. 

I assume you understand given the instance variable fix below